config/v3_5_experimental/types/config_test.go

```diff
@@ -183,7 +182,54 @@ func TestConfigValidation(t *testing.T) {
 			out: errors.ErrPathConflictsSystemd,
 			at:  path.New("json", "storage", "links", 0, "path"),
 		},
-		// test 6: non-conflicting scenarios
+
+		// test 6: file path conflicts with another file path, should error
+		{
+			in: Config{
+				Storage: Storage{
+					Files: []File{
+						{Node: Node{Path: "C:\\foo\\bar"}},
```

Comment:
That looks weird. Why are we validating this with Windows style paths here? This is only about the final system which Linux only AFAIK.

Reply:
Thanks for pointing that out. 
I was having CI failures on Windows, so I created tests to see if my validations applied to Windows as well. They're now working on Windows, but the unit tests should indeed be exclusive to Linux.